pkg/services/pluginsintegration/clientmiddleware/clear_auth_headers_middleware_test.go

```diff
@@ -157,6 +235,39 @@ func TestClearAuthHeadersMiddleware(t *testing.T) {
 				require.Len(t, cdt.CheckHealthReq.Headers, 1)
 				require.Equal(t, "test", cdt.CheckHealthReq.Headers[otherHeader])
 			})
+
+			t.Run("Should clear auth headers when calling SubscribeStream", func(t *testing.T) {
+				_, err = cdt.MiddlewareHandler.SubscribeStream(req.Context(), &backend.SubscribeStreamRequest{
+					PluginContext: pluginCtx,
+					Headers:       map[string]string{otherHeader: "test"},
```

Comment:
Stupid question, might be because how Github presents the diffs, but shouldn't we pass auth headers in the test case for the middleware to clear?

Reply:
Aha yes I knew I forgot that! This test file had me stumped initially as it wasn't clear how it was testing things - forgot to patch this up. Thanks for the careful review ❤️